controllers/devworkspace/solver/che_routing.go

```diff
@@ -505,7 +505,7 @@ func provisionMasterRouting(cheCluster *v2alpha1.CheCluster, routing *dwo.DevWor
 	return nil
 }
 
-func addEndpointToTraefikConfig(componentName string, e dw.Endpoint, cfg *traefikConfig, cheCluster *v2alpha1.CheCluster, routing *dwo.DevWorkspaceRouting) {
+func addEndpointToTraefikConfig(componentName string, e dw.Endpoint, cfg *gateway.TraefikConfig, cheCluster *v2alpha1.CheCluster, routing *dwo.DevWorkspaceRouting) {
```

Comment:
With a little bit more massaging I think this could be made to make use of the new helpers in `traefik_config_util.go`.

Reply:
same as above, I didn't want to change the workspace routing